src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api.UnitTests/Controllers/LocationsControllerTests.cs

```diff
@@ -0,0 +1,47 @@
+﻿using AutoFixture.NUnit3;
+using FluentAssertions;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SFA.DAS.ApprenticeAan.Api.Controllers;
+using SFA.DAS.ApprenticeAan.Application.Locations.Queries.GetAddresses;
+using SFA.DAS.Testing.AutoFixture;
+
+namespace SFA.DAS.ApprenticeAan.Api.UnitTests.Controllers;
+
+public class LocationsControllerTests
+{
+    [Test]
+    [MoqAutoData]
+    public async Task GetAddresses_ReturnsAddresses(
+        GetAddressesQueryResult response,
+        [Frozen] Mock<IMediator> mockMediator,
```

Comment:
The mockMediator and sut parameters could go into a ctor as seen [here](https://github.com/SkillsFundingAgency/das-aan-hub-api/blob/b5edde3a8e2081e9738a6cad615e9869da408d03/src/SFA.DAS.AANHub.Api.UnitTests/Controllers/AdminsControllerTests.cs#L30)

Reply:
@ravihejmadi The admin controller test is wrong, it cannot be in ctor, it has to be in a SetUp method. Unlike in XUnit, NUnit creates a single instance of a class for the tests, which means any class level properties initialized in constructor will share the setup in all the tests.